server/pom.xml

```diff
@@ -49,12 +49,6 @@
             <version>${project.parent.version}</version>
             <scope>runtime</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.druid</groupId>
-            <artifactId>druid-console</artifactId>
-            <version>${project.parent.version}</version>
-            <scope>runtime</scope>
-        </dependency>
```

Comment:
Heads up: I am reverting this change here https://github.com/apache/druid/pull/12489/commits/592ddd4ef05617968cd4c3a9be846933455db67b

Removing the druid-console dependency makes the web console no longer be part of Druid...

![image](https://user-images.githubusercontent.com/177816/166409119-8900a413-5a70-49f9-a1ed-fcb85b76e027.png)

The e2e tests do not run unless there are JS changes which is how this PR passed CI. I was struggling to figure out why my console PR (https://github.com/apache/druid/pull/12489) was failing to even start the e2e tests when it turned out that the console was just gone...

<img width="507" alt="image" src="https://user-images.githubusercontent.com/177816/166409381-a40b310d-0c80-478b-8628-6691ac5f69d3.png">

So I am adding this dependency back. Fun fact, my PR (liked above) will actually improve the console build speed a little bit by removing the `stylus` dependency but obviously not as much as not building the console all together.

Reply:
Based on ^, I think maybe we should consider running web e2e tests on java changes, instead of just web-console changes.

Reply:
Running web e2e tests on java changes may be too burden. IMO, we can add one test case to check if `localhost:888/unified-console.html` returns HTTP 200.

Reply:
@vogievetsky Do you mean the `druid-console.jar`  is missing in the distribution package after deleting the depedency declaration from server/pom.xml ?

Reply:
The final distribution package is packaged by maven-assembly-plugin. I think we can make some modifications to the assembly file under `distribution/src/assembly/assembly.xml` to include the `druid-console` module in the final package. If it's viable, we don't need to include `druid-console` module in the `druid-server` module which really looks a little bit strange.